src/main/java/org/jabref/gui/importer/ImportEntriesDialog.css

```diff
@@ -8,18 +8,13 @@
 }
 
 .entry-container {
-    /*-fx-padding: 0.5em 0em 0.5em 0em;*/
+    -fx-padding: 0.5em 0em 0.5em 0em;
 }
 
-.list-cell .summary > Text {
-    -fx-padding: 0.5em 0 1em 0.5em;
-    -fx-fill: -js-summary-text-color;
+.list-cell:default {
+    -fx-padding: 0.5em 0.1em 0.5em 0em;
 }
 
-.list-cell:entry-selected {
-    -fx-background-color: derive(-jr-selected, 35%);
-}
-
-.list-cell:entry-selected .summary > Text {
-    -fx-fill: js-summary-text-color-selected;
+.list-cell:selected {
```

Comment:
Im not sure, but i think this will also affect the dropdown menu to select the library to import to. Better select the import list by id and then select in the hierarchy below.

Reply:
I just removed them, and it looks like it's all good.